Doc/library/mailbox.rst

```diff
@@ -447,8 +447,9 @@ Supported mailbox formats are Maildir, mbox, MH, Babyl, and MMDF.
    is a callable object that accepts a file-like message representation (which
    behaves as if opened in binary mode) and returns a custom representation. If
    *factory* is ``None``, :class:`mboxMessage` is used as the default message
-   representation. If *create* is ``True``, the mailbox is created if it does not
-   exist.
+   representation (but this is not the same as setting *factory* to
```

Comment:
If it's not the same, then what is the difference? I saw the example in the bpo, but I feel like we should explain what happens in the documentation too.